modules/specifying-role-arn-albo-sts.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/installing-albo-sts-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-role-arn-albo-sts_{context}"]
+= Specifying the role ARN for ALBO on an STS cluster
```

Comment:
Can we use the term `ALBO`?

Reply:
I believe so, but I'll double-check that.

Reply:
Checked with my content strategist and we agree that the general suggestion is not to use such acronyms, but this particular section heading is already fairly long with the ALBO acronym in place. I'll leave it for now and see if it's an issue when it's time for peer review on this PR.

Reply:
https://docs.openshift.com/container-platform/4.14/networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.html, it seems like the acronym part should be ALB Operator. 

I also don't find it on this sheet, so maybe @kowen-rh its worth seeing about adding it here: https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1375785039&range=A150